tests/checks/test_geometry_hierarchy.py

```diff
@@ -0,0 +1,30 @@
+from geopandas import GeoDataFrame
+
+from src.checks.geometry_hierarchy import main
+
+
+def test_mdg_geometry_hierarchy(iso3: str, gdfs: list[GeoDataFrame]) -> None:
```

Comment:
I think this is likely to be a slow test due to the spatial join, right? How do you feel about making a Pytest mark for slow tests, such that they can be skipped if running tests for a feature that doesn't touch this? (I'll caveat that we should run all tests before committing, merging, etc. But I'm thinking of making the DX better during other feature development)

Reply:
Another good catch! I added `@pytest.mark.slow` to all the geometry tests.